frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt

```diff
@@ -87,6 +100,77 @@ class OverrideForkPointDialog(
             thisRowComment
           )
       }
+
+      row {
+        radioButton(
+          format(
+            getString(
+              "action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.radio-button.custom"
+            ),
+            radioButtonComment
+          ),
+          OverrideOption.CUSTOM
+        )
+
+        comboBox<ICommitOfManagedBranch?>(
+          (listOf(branch.forkPoint, parentBranch.pointedCommit) + branch.commits.toMutableList()).filterNotNull(),
```

Comment:
Hmmm there's a problem with taking `branch.commits` as a range of commits here, as `commits` already take existing fork point override into account (if one exists 😅)

As a quick workaround, I'd:
* add `commitsUntilParent` alongside `commits` to `NonRootManagedBranchSnapshot`, computed as `gitCoreRepository.deriveCommitRange(corePointedCommit, parentCoreLocalBranch.getPointedCommit())` regardless of `syncToParentStatus`
* rename `commits` to `uniqueCommits`
* use `branch.commitsUntilParent` instead of `branch.(unique)commits`

Reply:
(preferably on a PR under the current one)

Reply:
done here: https://github.com/VirtusLab/git-machete-intellij-plugin/pull/1273